CleanCode.SolidPrinciples/SingleResponsibility/ReferenceUtils.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace CleanCode.SolidPrinciples.SingleResponsibility
+{
+    public class ReferenceUtils
```

Comment:
Aktualnie tak, ale gdyby doszła jakaś nowa funkcjonalność związania z References to nazwa byłaby już niewłaściwa i dlatego zachowałem bardziej ogólne nazewnictwo